include/boost/property_tree/json_parser/detail/parser.hpp

```diff
@@ -5,6 +5,7 @@
 
 #include <boost/core/ref.hpp>
 #include <boost/bind/bind.hpp>
+#include <boost/bind/placeholders.hpp>
```

Comment:
@pdimov, I've initially added this header to address https://github.com/boostorg/property_tree/pull/101, but it looked incorrect on second thought. I believe it should be the user's responsibility to manage the definition of `BOOST_BIND_NO_PLACEHOLDERS` and it has nothing to with including `boost/bind/placeholders.hpp`. what do you think?